change/@fluentui-react-native-text-2020-05-20-11-46-47-textupdates.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Any idea why there are two change files for text?

Reply:
No idea :/

Reply:
I don't think it should matter, hopefully beachball is smart enough to resolve them together